app/services/adapters/webhook_destination.rb

```diff
@@ -0,0 +1,26 @@
+module Adapters
+  class WebhookDestination
+
+    class DestinationRequestError < StandardError
+    end
```

Comment:
Given the namespacing, wonder if we can rename this to something more simple:
Adapters:: WebhookDestination::ClientRequestError